frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/BrowseNavSection.tsx

```diff
@@ -83,6 +87,34 @@ export const BrowseNavSection = ({
           {t`Databases`}
         </PaddedSidebarLink>
       )}
+
+      <DelayedLoadingAndErrorWrapper
+        loading={areMetricsLoading}
+        error={metricsError}
+        loader={<SidebarSkeleton />}
```

Comment:
On reflection I think the lone skeleton is a little awkward. Might make sense to simply hide both the Browse models and Browse metrics links while they are loading. No loading indicator needed. This is the approach that Notion takes with its sidebar.

Reply:
@romeovs we are planning to show metrics and models pages even when there are no such items in the instance. We have an empty state for Metrics in this project. And there is a separate project about empty state for Models page.

Reply:
Ok let's keep this as is for now then and fix it when we work on that project.